src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListCommand.cs

```diff
@@ -208,13 +212,23 @@ public FeedbackInfo(string description, FeedbackType feedbackType, string summar
                 [OtherResultCommandId] = new FeedbackInfo(Resources.OtherResult, FeedbackType.OtherResult, Resources.OtherSummary),
             });
 
-        private static void DisplayFeedbackDialog(int commandId, SarifErrorListItem sarifErrorListItem)
+        private static void DisplayFeedbackDialog(int commandId, IEnumerable<SarifErrorListItem> sarifErrorListItems)
         {
             FeedbackInfo feedbackInfo = s_commandToResultDescriptionDictionary[commandId];
             string title = string.Format(Resources.ReportResultTitle, feedbackInfo.Description);
-            string summary = string.Format(feedbackInfo.Summary, sarifErrorListItem.Tool.Name, sarifErrorListItem.Rule.Id);
-            IEnumerable<string> snippets = sarifErrorListItem.GetCodeSnippets();
-            var feedbackDialog = new FeedbackDialog(title, sarifErrorListItem, feedbackInfo.FeedbackType, snippets, summary);
+            SarifErrorListItem item = sarifErrorListItems.FirstOrDefault();
+            string summary = string.Format(feedbackInfo.Summary, item.Tool.Name, item.Rule.Id);
+
+            var feedback = new FeedbackModel(
+                item.Rule.Id,
```

Comment:
>item.Rule.Id [](start = 16, length = 12)

same thing here, 2 results, 2 different rules. one rule in the feedback

Reply:
Combined RuleId/ToolNames/Versions if multiple values